components/cards/statement/statement.vue

```diff
@@ -109,6 +109,15 @@ export default class CardStatement extends mixins(
         return this.stores.cards.filteredStatement
     }
 
+    formatDate(val) {
+        const dateTime = DateTime.fromJSDate(val)
+        if (dateTime.hasSame(DateTime.now(), 'year')) {
+            return dateTime.toFormat('d MMMM')
+        }
+
+        return dateTime.toFormat('d MMMM yyyy')
+    }
+
     filterMonthChange(val) {
```

Comment:
Can this function also go to the baseComposable or util file?

Reply:
this will be done but not in this PR